### PR TITLE
Awesome-books Milestone 3: Complete web navigation

### PR DESCRIPTION
**Pull request by andres-condezo**
## Awesome books Milestone 3: Complete web navigation.

In this milestone we:
- Created the form and contact section, a navigation bar, and a footer in HTML.
- Created styles for the form and contact section, navigation bar, and footer.
- Created javaScript functions to display only one section at a time.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,26 +8,65 @@
...
+    <header id="header">
+      <nav>
+        <ul class="header__ul">
+          <li><a class="header__logo" href="#">Awesome Books</a></li>
+          <li class="desktop-menu-container">
+            <ul class="desktop-menu">
+              <li id='showList'><a href="#">List</a></li>
+              <li id='showAddBook'><a class="middle-item" href="#">Add new</a></li>
+              <li id='showContact'><a href="#">Contact</a></li>
+            </ul>
```
Comment (rloterh) [inline index.html]: - [x] To enhance the user experience of this design kindly ensure that the active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
image



**The wireframe menu design:**
image

File: src/navigation.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const showContact = document.querySelector('#showContact');
+
+const bookContainer = document.querySelector('#book-container');
+const addNew = document.querySelector('#add-new');
+const contactSection = document.querySelector('#contact-section');
+
+addNew.classList.add('hideSection');
+contactSection.classList.add('hideSection');
+
+function showPage(section) {
```
Comment (rloterh) [inline src/navigation.js]: - [x] OPTIONAL
It is nice to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

**Review by rloterh — APPROVED** [review]
Hi Jose & Andres, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Site navigation is working correctly
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andres-condezo** [comment]
## Hi @rloterh,
Thanks for the feedback, I will implement the changes :+1: .

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @andres-condezo and @joseheco , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**